root_consignment_export.tf

```diff
@@ -49,6 +49,7 @@ module "consignment_export_task_policy" {
       titleEnvironment          = title(local.environment),
       aws_region                = local.region,
       account                   = data.aws_caller_identity.current.account_id,
+      //TODO: change this argument in tdr-terraform-modules to accept decryption keys array (or upgrade to a component from da-terraform-modules) and pass through internal bucket key in addition
```

Comment:
@annielh Does migrating to the da-terraform-modules mean the entire ECS task is removed / re-created?